### DISCUSSION TITLE
KERI Implementer's Call 05/30/2024

### DISCUSSION THREAD

**Discussion by kentbull** [WebOfTrust/keri#86] [Meetings]
As Phil is on vacation this week I'll get this post started.

KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
WOT-Terms - @henkvancann

**Comment by kentbull** [comment]
SignifyTS:
One new PR since last week:
- [fix: delete contact by prefix #264](https://github.com/WebOfTrust/signify-ts/pull/264) - Fergal

Existing work:
- Open, failing PRs (on integration tests):
  - IPEX single sig [#234](https://github.com/WebOfTrust/signify-ts/pull/234) is in the process of being reviewed. - Fergal
  - AID rename and deletion [#241](https://github.com/WebOfTrust/signify-ts/pull/241) - Rodo
  - Adding integration test for multisig join: [#258](https://github.com/WebOfTrust/signify-ts/pull/258)
- Using Dev version of KERIA breaks tests
  - Draft: Use development version of KERIA [#252](https://github.com/WebOfTrust/signify-ts/pull/252) - Daniel
- Blocked on NPM publishing key/token - no longer blocked?
  - NPM package publishing PR [#260](https://github.com/WebOfTrust/signify-ts/pull/260) - Daniel
- Merge Conflicts:
  -  signature-params not serialized [#231](https://github.com/WebOfTrust/signify-ts/issues/231) - Provenant da-tychinin's

- Closeable issues:
  - [#227](https://github.com/WebOfTrust/signify-ts/issues/227) Multisig join never resolves. 
    - Fixed as of KERIA [#236](https://github.com/WebOfTrust/keria/pull/236)

CESRide:
- All PRs are over four three months old. We need a maintainer.
  - Charles's PRs:
    - [#166](https://github.com/WebOfTrust/cesride/pull/166/files) Creder -> SerderACDC
    - [#167](https://github.com/WebOfTrust/cesride/pull/167) **really small** PR to use non-deprecated Rust functions.
    - [#165](https://github.com/WebOfTrust/cesride/pull/167) Remove saider add wasm
    - Draft [#161](https://github.com/WebOfTrust/cesride/pull/161) Update code tables to CESR 2.0

**Reply by 2byrds** [reply]
KERIA - [Delegation approval PR](https://github.com/WebOfTrust/keria/pull/248) adds a new endpoint for approving anchored delegations to fix #230 
It is currently pointing to KERI main and will transition to a dev tag when available.

**Reply by iFergal** [reply]
If there's time, 3 issues I'd like to ask about:
- https://github.com/WebOfTrust/keria/issues/184 (this is actually more keripy related)
- https://github.com/WebOfTrust/keria/issues/232
- https://github.com/WebOfTrust/keria/issues/244 - agreed to not auto migrate but it's not clear to me if I should need to migrate or not with KERIA only now

**Comment by pfeairheller** [comment]
I am no longer on vacation and will be at tomorrow's meeting!!

SigPY:
- No new activity

**Comment by m00sey** [comment]
keripy:

Three PRs
- https://github.com/WebOfTrust/keripy/pull/789
- https://github.com/WebOfTrust/keripy/pull/788
- https://github.com/WebOfTrust/keripy/pull/790

**Comment by henkvancann** [comment]
Kerific - @kordwarshuis
WOT-Terms - @henkvancann
- No new activity
